projects/igniteui-angular/src/lib/grids/filtering/excel-style/excel-style-default-expression.component.ts

```diff
@@ -11,14 +11,15 @@ import {
 import { IgxColumnComponent } from '../../columns/column.component';
 import { ExpressionUI } from '../grid-filtering.service';
 import { IgxButtonGroupComponent } from '../../../buttonGroup/buttonGroup.component';
-import { IgxDropDownItemComponent, IgxDropDownComponent } from '../../../drop-down/public_api';
-import { IgxInputGroupComponent, IgxInputDirective } from '../../../input-group/public_api';
 import { DataType, DataUtil } from '../../../data-operations/data-util';
 import { IFilteringOperation } from '../../../data-operations/filtering-condition';
-import { OverlaySettings, ConnectedPositioningStrategy, CloseScrollStrategy } from '../../../services/public_api';
+import { OverlaySettings, ConnectedPositioningStrategy, AbsoluteScrollStrategy  } from '../../../services/public_api';
 import { KEYS, IBaseEventArgs } from '../../../core/utils';
 import { FilteringLogic } from '../../../data-operations/filtering-expression.interface';
 import { DisplayDensity } from '../../../core/density';
+import { IgxSelectComponent } from '../../../select/select.component';
```

Comment:
Regarding latest import changes, I believe we should import from the available public_api files (where available)

import { IgxSelectComponent } from '../../../select/public_api';